File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/anagrafiche/RubricaVoci.ts
import { Rubrica } from './Rubrica';

export class RubricaVoci {
    rubrica: Rubrica;
    valore: string;
    flag_cancellazione: boolean;

    constructor() {
        this.flag_cancellazione = false;
    }
}

export class RubricaVociConChiave extends RubricaVoci {
    chiave: number;
}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/anagrafiche/RubricaVoci.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.anagrafiche
{
    public class RubricaVoci
    {

        public Rubrica rubrica { get; set; }

        public string valore { get; set; }
        public bool flag_cancellazione { get; set; }

        public RubricaVoci()
        {
            this.flag_cancellazione = false;
        }
    }
}
